client/lib/application/test/access-form.test.jsx

```diff
@@ -25,25 +28,30 @@ APP_KIO = {
 
 describe('The access control form view', () => {
     var actionSpy,
```

Comment:
Since we are already using ES6 we could use `let` or `const` here

Reply:
done